script.js

```diff
@@ -20,3 +95,81 @@ document.querySelector('.menu').addEventListener('click', () => {
     closeButton.remove();
   });
 });
+
+function createpopups() {
+  const button = document.querySelectorAll('.card .button');
+  const buttonArr = Array.from(button);
+  buttonArr.map((el, i) => {
+    el.addEventListener('click', () => {
+      const popupContainer = document.createElement('div');
+      popupContainer.classList.add('popup-container');
+
+      const popup = document.createElement('div');
+      popup.classList.add('popup');
+
+      popupContainer.appendChild(popup);
+
+      const closeButton = document.createElement('a');
+      closeButton.classList.add('close-button');
+      closeButton.addEventListener('click', () => {
+        popupContainer.remove();
+      });
+      popup.appendChild(closeButton);
+
+      const img = document.createElement('img');
+      img.setAttribute('src', projects[i].img);
+      popup.appendChild(img);
+
+      const info = document.createElement('div');
+      info.classList.add('flexrow');
+
+      const txtinfo = document.createElement('div');
+      txtinfo.classList.add('flexcolumn');
+
+      const title = document.createElement('h2');
+      title.innerHTML = projects[i].title;
+      txtinfo.appendChild(title);
+
+      const tags = document.createElement('ul');
+      tags.classList.add('languages');
+      projects[i].tags.map((t) => {
+        const tag = document.createElement('li');
+        tag.innerHTML = t;
+        tags.appendChild(tag);
+        return null;
+      });
+      txtinfo.appendChild(tags);
+
+      const seeProjectButton = document.createElement('a');
+      seeProjectButton.classList.add('button');
+      seeProjectButton.classList.add('see-project-button');
+      seeProjectButton.innerHTML = 'See Project';
+      info.appendChild(seeProjectButton);
+
+      const seeSourceButton = document.createElement('a');
+      seeSourceButton.classList.add('button');
+      seeSourceButton.classList.add('see-source-button');
+      seeSourceButton.innerHTML = 'See Source';
+      info.appendChild(seeSourceButton);
```

Comment:
I believe these buttons' appearance can be improved to look more like those in the figma design as a project requirement. Please checkout the comparison below 👇🏻 and make some changes on the right padding to give space to the icon and the button background color.

| Your Button | Figma Button |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/151230671-80a1a396-668d-40cd-becd-e8a3d184e867.png) | ![image](https://user-images.githubusercontent.com/4247740/151231085-f569c2dd-67e3-472b-87bd-38db8eb4dbb8.png) |

Reply:
The color of the buttons is due to accessibility issues with the originals, apart from that, will do 🎉